esx_service/vmdk_ops.py

```diff
@@ -166,7 +166,9 @@ def createVMDK(vmdk_path, vm_name, vol_name, opts={}, vm_uuid=None, tenant_uuid=
     logging.info("*** createVMDK: %s opts = %s", vmdk_path, opts)
 
     if os.path.isfile(vmdk_path):
-        return err("File %s already exists" % vmdk_path)
+        # We are mostly here due to race or Plugin VMCI retry #1076
+        logging.warning("File %s already exists", vmdk_path)
+        return None
```

Comment:
Filed a new issue #1089. Yes, it's a rare case where Volume Create is requested from 2 separate nodes at the same time. Addressing it would require some code cleanup since some default options are set in KV (diskformat, fstype) but some are not set (e.g. size).
@govint Are you okay addressing this in separate issue?